app/models/account.rb

```diff
@@ -513,7 +513,8 @@ def advanced_search_for_sql_template(following)
         <<-SQL.squish
           SELECT
             accounts.*,
-            (count(f.id) + 1) * #{BOOST} * ts_rank_cd(#{TEXTSEARCH}, to_tsquery('simple', :tsquery), 32) AS rank
+            (count(f.id) + 1) * #{BOOST} * ts_rank_cd(#{TEXTSEARCH}, to_tsquery('simple', :tsquery), 32) AS rank,
```

Comment:
The count no longer needs to be in this calculation.

Reply:
So you mean the line should change to ` #{BOOST} * ts_rank_cd(#{TEXTSEARCH}, to_tsquery('simple', :tsquery), 32) AS ran` ?

Reply:
Yes

Reply:
Resolved in 79639cd6177d709eab2a38d79b825f696f6e12c8